src/Microsoft.AspNetCore.Sockets.Client/IChannelConnection.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
If we did Sockets.Common we could at least move this there until it is accepted into Channels. We should probably just submit a PR to corefxlab